Create Pages “how-to-guide-for-covid-safer-events-1” (#21)
breathe-easy-events/sheffield.breathe-easy.uk: src/pages/how-to-guide-for-covid-safer-events-1.md

---
title: How-to guide for Covid-safer events
menu: true
layout: ResourceLayout.11ty.tsx
discription: A guide for anyone who organises events indoors and wants to keep
  attendees safe(r) from the life-altering impacts of Covid-19 and other
  airborne diseases.
---
A guide for anyone who organises events indoors and wants to keep attendees safe(r) from the life-altering impacts of Covid-19 and other airborne diseases.

## 🔬 Choosing your safety measures

> Thank-you for thinking about ways to make people safer at your event. 
> 
> There are seven Covid safety measures in this guide. Many are cheap or free, although some require upfront investment and some take more time.

Author: ```suggestion
Thank-you for thinking about ways to make people safer at your event. 

don't do bad thing here
```
